gcloud/logging/logger.py

```diff
@@ -89,10 +113,15 @@ def log_text(self, text, client=None):
                 },
             }],
         }
+
+        labels = self._get_labels(labels)
+        if labels is not None:
+            data['entries'][0]['labels'] = labels
```

Comment:
Add a comment here or use an intermediate variable? Using `__getitem__` three times in a row hides a bit.